.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
Are you planning to add 3.14 in a follow-up PR? While we're touching this repository, I'd love to get this matrix fully up to date.

Reply:
We could add 3.14, but it's in alpha right now - it looks like the proper release would be next year: https://peps.python.org/pep-0745/. Do we want to add prerelease versions to our matrix ahead of time?

Reply:
Chatted offline, leaving 3.14 out until it's released!